modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -1,122 +1,82 @@
-
-=== Prerequisites
-
-. https://redpanda.com/try-redpanda?section=cloud-trial[Sign up for a beta trial^] of Redpanda Serverless.
-
-. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
-
-=== Credits and limits
-
-Each beta trial supports five Serverless clusters. Redpanda issues 100 free credits, which is enough to run Redpanda with reasonable throughput. These credits expire in 14 days. Usage is based on the following:
-
-* Ingress: 0.1 credit per GB in (writes)
-* Egress: 0.1 credit per GB out (reads)
-* Partitions: 0.003 credit (metered in 10 min fractions) per partition/hour	
-* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/month
-
-After either the 14 day trial expires or the 100 credits expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial, in case you want to extend it. After that, the data is permanently deleted. To extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
+[NOTE]
+====
+These baseline limits are subject to change. To increase your limits, contact https://support.redpanda.com/hc/en-us/requests/new[support^]. 
+====
 
-=== Get started
+== Trial credits
 
-For the beta release, you'll receive an email to activate your Serverless trial. 
+New trials receive $100 (USD) in free credits to spend in the first 14 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required. To continue using Serverless after your trial expires, you can enter a credit card and pay as you go. Any remaining credit balance is used before you are charged. 
 
-. In your Welcome email, click to sign up.
-+
-Redpanda prepares a Redpanda Cloud account for you with a "Hello, World!" Serverless cluster in AWS.
+When either the 100 credits expire or the 14 days in the trial expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 7 day grace period following the end of the trial. After that, the data is permanently deleted. 
 
-. To log in to Redpanda Cloud, run the following command in your terminal:
-+
-```
-rpk cloud login
-```
+== Get started
 
-. Follow the CLI output for next steps.
+After you click to start a trial, Redpanda instantly prepares an account for you. Your account includes a `welcome` cluster with a `hello-world` demo topic you can explore. The `hello-world` topic is free as a read-only topic. It becomes billable with your credits if you produce messages to it. 
```

Comment:
No longer read only -- next sentence also contradicts the read only part. Maybe "starts as a free read-only topic, and is materialized and becomes billable ..."